__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require('supertest')
+const app = require('../app')
+const seed = require('../db/seeds/seed')
+const db = require('../db/connection')
+const testData = require('../db/data/test-data')
+
+beforeEach(() => seed(testData))
+afterAll(() => db.end())
+
+describe('GET /api/topics', () => {
+    it('returns a 200 status code and returns an array of topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((res) => {
+            console.log(res.body.topics)
```

Comment:
Please make sure to remove any stray console.logs() before comitting and pushing - whenever you are creating a PR, that means you are completely finished with this piece of work, which means you should have no stray console.logs or commented out code around!